Sources/SentrySwiftUI/SentryInternal/SentryInternal.h

```diff
@@ -11,6 +11,7 @@ NS_ASSUME_NONNULL_BEGIN
 
 typedef NS_ENUM(NSInteger, SentryTransactionNameSource);
 
+@class UIView;
```

Comment:
I don't think this is needed, I was able to build iOS-Swift and iOS-SwiftUI without it, as well as Sentry for the uikitless config, and I worry about what it would do for our UIKit linking setup.

```suggestion
```

Reply:
Oh yeah, I removed what was using it. Thanks!!